{"name": "test_completed_counter_increases_on_new_order[firefox]", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'setup' for <Function test_open_order_details_popup[firefox]>>\n\n    @pytest.fixture(params=[\"chrome\", \"firefox\"], scope=\"class\")\n    def setup(request):\n>       driver = initialize_driver(request.param)\n\nconftest.py:6: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nbrowser_name = 'firefox'\n\n    def initialize_driver(browser_name):\n        if browser_name == \"chrome\":\n            driver = webdriver.Chrome(ChromeDriverManager().install())\n        elif browser_name == \"firefox\":\n>           driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\nE           TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nutils\\driver_setup.py:9: TypeError"}, "parameters": [{"name": "setup", "value": "'firefox'"}], "start": 1730625013516, "stop": 1730625013516, "uuid": "ddbd7ca4-5aad-4243-a0ed-cce404ba0d9d", "historyId": "563062db10758a479c2bc89593f0fa6f", "testCaseId": "80b884980d95fc6ee4c51e2046dfb80b", "fullName": "tests.test_order_feed.TestOrderFeed#test_completed_counter_increases_on_new_order", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "9892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}